src/core_ct/slice.py

```diff
@@ -32,15 +32,15 @@ def __init__(self, data: np.ndarray, pixel_dimensions: tuple[float, float]):
         self.data: np.ndarray = data
         self.pixel_dimensions: tuple[float, float] = pixel_dimensions
 
-    def trim(self, axis: int, loc_start: int, loc_end: int | None = None) -> Slice:
+    def trim(self, axis: int, left_index: int, right_index: int | None = None) -> Slice:
         """
         Create new slice by trimming off a specified amount on the requested axis.
 
         Remove unwanted space around a core that must be removed before analysis.
```

Comment:
I'm not sure how useful this line is, since `trim()` is meant to be used for a wide variety of things. Maybe there should be a "potential use cases" section where this would fit better.